static/app/components/replays/player/replayPlayerContainment.tsx

```diff
@@ -0,0 +1,84 @@
+import type {CSSProperties, ReactNode} from 'react';
+import {useRef} from 'react';
+import {css} from '@emotion/react';
+
+import divide from 'sentry/utils/number/divide';
+import useReplayPlayerState from 'sentry/utils/replays/playback/providers/useReplayPlayerState';
+import {useDimensions} from 'sentry/utils/useDimensions';
+
+interface Props {
+  /**
+   * You must pass `styles` into the <ReplayPlayer>,
+   */
+  children: (styles: CSSProperties) => ReactNode;
+
+  /**
+   * How to measure and resize the player.
+   *
+   * "both"
+   *    The default. Measure the player fully. Resize the player to fit within
+   *    the given width & height. Useful if you want to reserve some space for
+   *    the replay.
+   *    ie: with a wrapper that has CSS `height` set.
```

Comment:
should we add a note about which one (`both` vs `width`) is used most often (similar to the note you made in the story)?

Reply:
hrm, i probably shouldn't assume what's more common, but better explain the context for picking each.

the `width` option is good for the replay page itself because the height is flexible.